mkt/abuse/forms.py

```diff
@@ -0,0 +1,60 @@
+from django import forms
+from django.forms.models import modelformset_factory
+
+import happyforms
+from quieter_formset.formset import BaseModelFormSet
+
+import mkt
+from mkt.reviewers.models import ReviewerScore
+from mkt.webapps.models import Webapp
+
+
+class BaseAbuseViewFormSet(BaseModelFormSet):
+
+    def __init__(self, *args, **kwargs):
+        self.form = AbuseViewForm
+        self.request = kwargs.pop('request', None)
+        super(BaseAbuseViewFormSet, self).__init__(*args, **kwargs)
+
+    def save(self):
+        for form in self.forms:
```

Comment:
I do it in the view method before calling save()
https://github.com/mozilla/zamboni/pull/3078/files#diff-9a26e6efe2570209f70b06294f23ca39R625